azure/notebooks/kubernetes/Dask_Kube_XGBoost.ipynb

```diff
@@ -0,0 +1,917 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    print(nyc_tlc_df.head()) # does not work without pandas 1.0.0 which gets installed with azure-opendatasets
</pre><p>We can remove mention of the Pandas update again since we discussed at the top</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Good catch.</p>